### PR TITLE
exporter: add jmx-exporter for yarn

### LINKED ISSUE #230: exporter:  add jmx-exporter for yarn

### PR DESCRIPTION
**Pull request by nolimitkun**
#### Which issue(s) this PR fixes:

Fixes #230 

#### Additional comments:

To make clear that you license your contribution under
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0) and that you give permission to [TOSIT](https://www.tosit.io/),
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

**Review by rpignolet — APPROVED** [review]

File: playbooks/exporter_jmx_install.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 ---
 - name: jmx-exporter install
-  hosts: hdfs_nn, hdfs_jn, hdfs_dn
+  hosts: hdfs_nn, hdfs_jn, hdfs_dn, yarn_rm, yarn_nm, yarn_ats, mapred_jhs
```
Comment (gboutry) [inline playbooks/exporter_jmx_install.yml:6]:
```suggestion
  hosts: hdfs_nn:hdfs_jn:hdfs_dn:yarn_rm:yarn_nm:yarn_ats:mapred_jhs
```

We're using columns when specifying multiple hosts in other playbooks, we should keep the same characeter

File: playbooks/yarn.yml [hunk 2]
```diff
@@ -18,6 +18,7 @@
 - import_playbook: yarn_ranger_config.yml
 
 - import_playbook: yarn_ranger_init.yml
+- import_playbook: yarn_jmx-exporter_config.yml
```
Comment (gboutry) [inline playbooks/yarn.yml:21]:
```suggestion
- import_playbook: yarn_ranger_config.yml
- import_playbook: yarn_jmx-exporter_config.yml

- import_playbook: yarn_ranger_init.yml
```

A config task should stay in a config block if possible

File: playbooks/exporter_jmx_install.yml (hunk 1 above)
Comment (nolimitkun) [reply playbooks/exporter_jmx_install.yml]: Yep, done.

File: playbooks/yarn.yml (hunk 2 above)
Comment (nolimitkun) [reply playbooks/yarn.yml]: done.

**Review by gboutry — APPROVED** [review]

**Pull request merged**